modules/db/src/main/scala/gem/dao/DatasetDao.scala

```diff
@@ -7,30 +7,48 @@ import doobie.imports._
 import java.time.Instant
 
 object DatasetDao {
```

Comment:
Since the DAOs don't do much other than delegate to `Statements`, does it make sense to just get rid of them and call `.run`, `.list` etc. from the places where the statements are used?